pkg/cli/command/list-plugins.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2019-present Open Networking Foundation.
```

Comment:
This doesn't have to be done now, but I would suggest to put both commands into 'plugins.go' (or models.go) and then structure the commands for 'onos plugins {list|load} usage' or perhaps 'onos models {list|load}'